src/main/resources/application.conf

```diff
@@ -9,6 +9,8 @@ conseil: {
 # Runtime settings for Lorre process
 lorre: {
   sleepInterval: 60 s
+  retryInterval: 10 s
```

Comment:
Can we give these more specific names like `bootupRetryInterval` and `bootupConnectionCheckTimeout`?

Reply:
ok, changed